framework/db/ActiveRecord.php

```diff
@@ -114,7 +114,13 @@ class ActiveRecord extends BaseActiveRecord
      */
     public function loadDefaultValues($skipIfSet = true)
     {
-        foreach ($this->getTableSchema()->columns as $column) {
+        static $columns = null;
+
+        if ($columns === null) {
+            $columns = $this->getTableSchema()->columns;
+        }
```

Comment:
Unlikely to male sense.
Here you avoiding just single call of `getTableSchema` by the cost of extra memory usage, which stores exact(!) copy of `TableSchema::columns`.